libraries/AP_Notify/examples/ToshibaLED_test/ToshibaLED_test.cpp

```diff
@@ -13,13 +14,18 @@ AP_Notify notify;
 
 static ToshibaLED_I2C toshiba_led(1);
 
+#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
+#include <SITL/SITL.h>
+SITL::SIM sitl;
+#endif
+
```

Comment:
I don't see why this is necessary here?

Reply:
The sitl object doesn't get instantiated anywhere else, and is used in <some codepath I can't recall but seemed reasonable>

We're trying to use the simulated i2c stuff in SITL and that code all assumes that AP::sitl() works.